lib/struct.js

```diff
@@ -184,6 +186,16 @@ function toString () {
 }
 
 /**
+ * Change struct padding mode.
+ *
+ * @api public
+ */
+function usePack (flag) {
```

Comment:
I don't see much value in having this be an instance method. Why would someone need to change the packed-ness during runtime? C doesn't support this so I'm not sure we need to.